rust/xaynet-client/src/mobile_client/participant/awaiting.rs

```diff
@@ -115,7 +115,7 @@ mod tests {
 
         match part.determine_role(eligible_sum_seed, 0.5_f64, 0.5_f64) {
             Role::Summer(_) => (),
-            _ => assert!(false),
+            _ => panic!(),
```

Comment:
maybe we can  use `assert!(matches!(...))` instead of `panic!()` here?

Reply:
good idea, i'll change that for all tests in that module.

it would be nice to have a negation of `if let`, that would make things even more clear.